proxy/src/main/java/org/apache/rocketmq/proxy/grpc/v2/DefaultGrpcMessingActivity.java

```diff
@@ -87,6 +87,7 @@ protected void init(MessagingProcessor messagingProcessor) {
         this.routeActivity = new RouteActivity(messagingProcessor, grpcClientSettingsManager, grpcChannelManager);
         this.clientActivity = new ClientActivity(messagingProcessor, grpcClientSettingsManager, grpcChannelManager);
 
+        this.appendShutdown(this.grpcChannelManager);
```

Comment:
Although the start method of grpcChannelManager is currently an empty implementation, it would be better to use appendStartAndShutdown.